docs/framework/interop/marshaling-data-with-platform-invoke.md

```diff
@@ -19,35 +19,35 @@ To call functions exported from an unmanaged library, a .NET Framework applicati
 
 - Substitute managed data types for unmanaged data types.
 
-You can use the documentation supplied with an unmanaged function to construct an equivalent managed prototype by applying the attribute with its optional fields and substituting managed data types for unmanaged types. For instructions about how to apply the **DllImportAttribute**, see [Consuming Unmanaged DLL Functions](../../../docs/framework/interop/consuming-unmanaged-dll-functions.md).
+You can use the documentation supplied with an unmanaged function to construct an equivalent managed prototype by applying the attribute with its optional fields and substituting managed data types for unmanaged types. For instructions about how to apply the `DllImportAttribute`, see [Consuming Unmanaged DLL Functions](../../../docs/framework/interop/consuming-unmanaged-dll-functions.md).
 
 This section provides samples that demonstrate how to create managed function prototypes for passing arguments to and receiving return values from functions exported by unmanaged libraries. The samples also demonstrate when to use the <xref:System.Runtime.InteropServices.MarshalAsAttribute> attribute and the <xref:System.Runtime.InteropServices.Marshal> class to explicitly marshal data.
 
 ## Platform invoke data types
 
 The following table lists data types used in the Win32 API (listed in Wtypes.h) and C-style functions. Many unmanaged libraries contain functions that pass these data types as parameters and return values. The third column lists the corresponding .NET Framework built-in value type or class that you use in managed code. In some cases, you can substitute a type of the same size for the type listed in the table.
 
-|Unmanaged type in Wtypes.h|Unmanaged C language type|Managed type name|Description|
+|Unmanaged type in Win32 APIs|Unmanaged C language type|Managed type|Description|
```

Comment:
Perhaps Windows APIs instead of Win32 APIs?

Reply:
I agree.

> Note that this was formerly called the Win32 API. The name Windows API more accurately reflects its roots in 16-bit Windows and its support on 64-bit Windows.

https://docs.microsoft.com/windows/desktop/apiindex/windows-api-list